services/worker-manager/schemas/v1/config-azure.yml

```diff
@@ -0,0 +1,157 @@
+      api method. worker-manager sets the values of the `osProfile.adminPassword`, `osProfile.computerName`, `osProfile.customData`, `networkProfile.networkInterfaces`
+      properties directly, and they cannot be overriden. It also fills in some basic labels, will generate a default description, and sets the `taskcluster`
+      metadata key directly. You are free to set other metadata items though. A subset of these fields we require you to set such as `machineType`.
+    uniqueItems: false
+    items:
+      type: object
+      properties:
+        capacityPerInstance:
+          title: Capacity Per Instance
+          type: integer
+          minimum: 1
+          description: The number of tasks a single worker of this type can run at any given time.
+        workerConfig:
+          title: Worker Config
+          type: object
+          additionalProperties: true
+          description: |
+            This value is supplied unchanged as the `workerConfig` property of the `taskcluster` instance metadata attribute.
+            The expectation is that the worker will merge this information with configuration from other sources,
+            and this is precisely what [taskcluster-worker-runner](https://github.com/taskcluster/taskcluster-worker-runner) does.
+            This property must not be used for secret configuration, as it is visible both in the worker pool configuration and in the worker instance's metadata.
+            Instead, put secret configuration in the [secrets service](https://github.com/taskcluster/taskcluster-worker-runner#secrets).
+        location:
+          title: Location
+          type: string
+          description: The Azure location this worker should be spawned in
+        subnetId:
+          title: Subnet ID
+          type: string
+          description: The ID of the subnet workers should be spawned in
+        priority:
+          title: Priority
+          type: string
+          description: |
+            virtual machine priority, i.e. Spot
+            https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/createorupdate#virtualmachineprioritytypes
+        billingProfile:
+          title: Billing Profile
+          type: object
+          description: https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/createorupdate#billingprofile
+          additionalProperties: true
+          required: []
+          properties:
+            maxPrice:
+              title: Max Price
+              type: number
+              description: the maximum price to pay for a spot virtual machine in US Dollars
+        hardwareProfile:
+          title: Hardware Profile
+          type: object
+          description: https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/createorupdate#hardwareprofile
+          additionalProperties: true
+          required: [vmSize]
```

Comment:
let's just do this the

```yaml
foo: 
  - bar
```
way to match up with how we do it everywhere else even if it uses more lines